cpan/Math-Complex/t/Trig.t

```diff
@@ -28,7 +30,7 @@ if ($^O eq 'unicos') { # See lib/Math/Complex.pm and t/lib/complex.t.
     $eps = 1e-10;
 }
 
-sub near ($$;$) {
```

Comment:
You can't use :prototype in a dual-life module.

Reply:
I agree with you on this, but this module is customized for the last few years on blead
as long as the `:prototype` part is not shipped to CPAN we are fine, no?